storybook/qmlTests/TestComponents/tst_StatusChatInput.qml

```diff
@@ -0,0 +1,79 @@
+import QtQuick 2.14
+import QtQml 2.14
+import QtTest 1.0
+
+import utils 1.0
+import shared.status 1.0
+import shared.stores 1.0
+
+Item {
+    id: root
+    width: 600
+    height: 400
+
+    QtObject {
+        id: globalUtilsMock
+
+        property var plainText
+        property var isCompressedPubKey: function (publicKey) {
+            return false
+        }
+    }
+
+    QtObject {
+        id: rootStoreMock
```

Comment:
It looks familiar to me :) . Maybe, as commented in your other pr, we can have mocks in a separate folder and in single files and reuse them in different contexts.

Reply:
Yeah, would be nice :)

Reply:
That's the plan. But probably not in this PR.

Reply:
It's true, wouldn't be good to add these changes in this PR. But I totally agree with you!